apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.ts

```diff
@@ -14,4 +14,10 @@ export class CardBarComponent {
 
     /** bar hex color */
     @Input() color = '#1866b4';
```

Comment:
You can't just create a component `fd-card-bar`. This would mean for the user that it's coming from Fundamental library.